src/get_ensemble_output.R

```diff
@@ -3,41 +3,69 @@ library(hubVis)
 library(hubEnsembles)
 library(hubData)
 library(dplyr)
+library(purrr)
```

Comment:
From the documentation: https://here.r-lib.org/#usage:

> The here package creates paths relative to the top-level directory. The package displays the top-level of the current project on load or any time you call [here()](https://here.r-lib.org/reference/here.html):
>
> ```r
> here::i_am("README.Rmd")
> #> here() starts at /home/kirill/git/R/here
> here()
> #> [1] "/home/kirill/git/R/here"
> ```

The key is to put `here::i_am("/path/to/current/file")` so that the here package knows where this file is relative to the hub. 

```suggestion
here::i_am("scr/get_ensemble_output.R")
library(purrr)
```